risc0/zkvm/verify/src/lib.rs

```diff
@@ -60,28 +60,37 @@ impl Receipt {
     }
 }
 
-// #[cfg(test)]
-// mod tests {
-//     use std::{fs, io};
+#[cfg(test)]
+mod tests {
+    use super::Receipt;
+    use std::vec::Vec;
+    use core::convert::TryFrom;
+    use std::fs;
+    use std::io;
+    use test_log::test;
 
-//     use arrayref::array_ref;
-//     use log::info;
-//     use test_log::test;
+    #[test]
+    fn test_receipt() -> io::Result<()> {
+        log::set_max_level(log::LevelFilter::Info);
+        let data: Vec<u8> = fs::read("src/simple_receipt.receipt")?;
```

Comment:
Can we do the annoying bazel stuff to actually generate this from the c++ rather than checking in a binary?  This probably involves making a specific new bin that generates a valid receipt and calling it as a bazel build step, which is annoying but 1) I don't like 100k binaries in git 2) It will make sure that the c++ and rust stay compatible moving forward.

Reply:
This stuff is all built and tested from cargo, so it's difficult to access bazel.  Maybe after @flaub's reworks it will be easier?

Reply:
Forgot about that... sigh, I guess binary it is.